<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-01-30T17:59:24 IST" name="Default suite" finished-at="2023-01-30T17:59:33 IST" duration-ms="9495">
    <groups>
    </groups>
    <test started-at="2023-01-30T17:59:24 IST" name="Default test" finished-at="2023-01-30T17:59:33 IST" duration-ms="9495">
      <class name="com.makemytrip.testscripts.BookmyticketTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.makemytrip.testscripts.BookmyticketTest@1f0f1111]" started-at="2023-01-30T17:59:24 IST" name="launchApplication" finished-at="2023-01-30T17:59:32 IST" duration-ms="8183" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="bookmyticket()[pri:0, instance:com.makemytrip.testscripts.BookmyticketTest@1f0f1111]" started-at="2023-01-30T17:59:32 IST" name="bookmyticket" finished-at="2023-01-30T17:59:32 IST" duration-ms="80" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at java.base/java.util.Objects.requireNonNull(Objects.java:208)
at org.openqa.selenium.interactions.Actions.<init>(Actions.java:65)
at com.makemytrip.pages.LandingPage.<init>(LandingPage.java:27)
at com.makemytrip.testscripts.BookmyticketTest.bookmyticket(BookmyticketTest.java:15)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bookmyticket -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.makemytrip.testscripts.BookmyticketTest@1f0f1111]" started-at="2023-01-30T17:59:32 IST" name="tearDown" finished-at="2023-01-30T17:59:33 IST" duration-ms="1167" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.makemytrip.testscripts.BookmyticketTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
